src/test/ui/mismatched_types/E0281.rs

```diff
@@ -1,25 +0,0 @@
-// Copyright 2016 The Rust Project Developers. See the COPYRIGHT
```

Comment:
If E0281 still exists, please write a test that reproduces it. That's the `Fn<usize>` test.